src/App.scss

```diff
@@ -1,34 +1,37 @@
-@import "./theme/colors.scss";
-@import "./theme/typography.scss";
+@use "./theme/colors.scss";
+@use "./theme/typography.scss";
```

Comment:
Use `@use` for namespaced imports via dart sass: https://sass-lang.com/documentation/at-rules/use

Reply:
brilliant